tests/software/fasttree/main.nf

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FASTTREE } from '../../../software/fasttree/main.nf' addParams( options: [:] )
+
+workflow test_fasttree {
+    
+    input = [ file(params.test_data['sarscov2']['genome']['alignment']['informative_sites_alignment'], checkIfExists: true) ]
```

Comment:
```suggestion
    input = [ file(params.test_data['sarscov2']['genome']['informative_sites_alignment'], checkIfExists: true) ]
```

Reply:
Its fine to go with a flatter structure because the file names will be unique anyway in these instances.